docs/testnet/onomy-testnet-docs/pre-installation.md

```diff
@@ -26,24 +25,26 @@ export PATH=$PATH:$HOME/.onomy/bin
 `
```

Comment:
Check markdown link check it is red
https://github.com/onomyprotocol/onomy/runs/4279388673?check_suite_focus=true

Reply:
Done